### PR TITLE
libvirt_vmxml: add func to check dumpxml

### PR DESCRIPTION
**Pull request by dzhengfy**
Signed-off-by: Dan Zheng <dzheng@redhat.com>

### CODE REVIEW

**Review by Yingshun — APPROVED** [review]
LGTM

File: virttest/utils_libvirt/libvirt_vmxml.py [hunk 1]
```diff
@@ -25,6 +29,26 @@ def set_vm_attrs(vmxml, vm_attrs):
...
+def check_guest_xml(vm_name, pat_in_dumpxml, option='', status_error=False):
+    """
+    Check the given pattern in the vm dumpxml
+
+    :param vm_name: vm name
+    :param pat_in_dumpxml:  str, the pattern to search in dumpxml
+    :param status_error: True if expect not existing, otherwise False
+    :raises: TestFail if the result is not expected
+    """
+    ret_stdout = virsh.dumpxml(vm_name, extra=option).stdout.strip()
```
Comment (chunfuwen) [inline virttest/utils_libvirt/libvirt_vmxml.py:41]: change 
stdout -->stdout_text ?

Comment (dzhengfy) [reply virttest/utils_libvirt/libvirt_vmxml.py:41]: @chunfuwen , when I check def command() in virsh.py, I find below in Line 812,813, so I think here 'stdout' is ok to use. How do you think of?
```
        ret.stdout = ret.stdout_text
        ret.stderr = ret.stderr_text
```

**Review by chunfuwen — APPROVED** [review]
LGTM

**Pull request merged**